components/link/stories/link.stories.js

```diff
@@ -70,5 +73,88 @@ export default {
 	},
 };
 
-export const Default = Template.bind({});
-Default.args = {};
+export const Default = (args, context) => html`
+	${window.isChromatic() ? ChromaticLinkGroup(args, context) : Template(args, context)}
+
+`;
+Default.storyName = "Primary";
```

Comment:
It makes sense to have "Primary" and "Secondary" variants in the Docs page, but it feels a little less logical when we only have the "Primary" story in the sidebar, do you think it would be worthwhile to have both "Default" and "Primary" so that we can use one in the sidebar and one on the Docs page?

Reply:
Yes! I've seen that in some other places now, too. Thanks for catching that- https://github.com/adobe/spectrum-css/pull/2884/commits/724b84fb0d5ecd5982c90c65f9523641f2b9b9a0